app/src/main/java/org/sopt/pingle/presentation/ui/main/plan/PlanRecruitmentFragment.kt

```diff
@@ -0,0 +1,81 @@
+package org.sopt.pingle.presentation.ui.main.plan
+
+import android.os.Bundle
+import android.view.KeyEvent
+import android.view.View
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.flowWithLifecycle
+import androidx.lifecycle.lifecycleScope
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import org.sopt.pingle.R
+import org.sopt.pingle.databinding.FragmentPlanRecruitmentBinding
+import org.sopt.pingle.util.base.BindingFragment
+import org.sopt.pingle.util.component.CustomSnackbar
+import org.sopt.pingle.util.context.hideKeyboard
+
+class PlanRecruitmentFragment :
+    BindingFragment<FragmentPlanRecruitmentBinding>(R.layout.fragment_plan_recruitment) {
+
+    private val viewModel: PlanViewModel by activityViewModels()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.planViewModel = viewModel
+        binding.lifecycleOwner = this
+
+        addListeners()
+        collectData()
+    }
+
+    private fun addListeners() {
+        with(binding) {
+            root.setOnClickListener {
+                requireActivity().hideKeyboard(it)
+                viewModel.setSelectedRecruitment(etPlanRecruitmentInputNumber.text.toString())
+            }
+
+            etPlanRecruitmentInputNumber.setOnKeyListener(
+                View.OnKeyListener { _, keyCode, event ->
+                    if (keyCode == KeyEvent.KEYCODE_ENTER && event.action == KeyEvent.ACTION_UP) {
+                        viewModel.setSelectedRecruitment(etPlanRecruitmentInputNumber.text.toString())
+                        return@OnKeyListener true
+                    }
+                    false
+                }
+            )
+
+            btnPlanRecruitmentPlus.setOnClickListener {
+                viewModel.incRecruitmentNum()
+            }
+
+            btnPlanRecruitmentMinus.setOnClickListener {
+                viewModel.decRecruitmentNum()
+            }
+        }
+    }
+
+    private fun collectData() {
+        viewModel.selectedRecruitment.flowWithLifecycle(lifecycle).onEach {
+            when (it.toString()) {
```

Comment:
```suggestion
        viewModel.selectedRecruitment.flowWithLifecycle(lifecycle).onEach { selectedRecruitment ->
            when (selectedRecruitment.toString()) {
```

가독성을 위해 이런 식으로 적어줍시다